Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -53,6 +53,9 @@ namespace itk
  *
  * GDCM build, instead of the one included within ITK itself.
  *
+ * The compressors supported include "JPEG2000" (default), and
+ * "JPEG". The compression level parameter is not supported.
```

Comment:
Technicaly GDCM does not do much here, simply forward parameter. So saying out loud 'compression level parameter is not supported' feels a bit wrong, maybe 'compression level parameter is not implemented' feels more accurate.

Reply:
Currently, this parameter is not being forwarded  and there is not compression quality forward hence the parameter is not supported.